src/services/WarehouseCrawlDataServices.ts

```diff
@@ -0,0 +1,65 @@
+import puppeteer from 'puppeteer';
+import winston from '../config/winston';
+
+import request_promise from 'request-promise';
+import cheerio from 'cheerio';
```

Comment:
sao lại dùng cả cheerio ?

Reply:
em dùng puppeteer mất quá nhiều tgian, nếu khách hàng bắt crawl tất cả 1 lúc toàn bộ kho theo các tỉnh em phải dùng them vòng for khi đó puppteer load rất lâu em sợ sẽ chết server. em dùng thử thấy thằng cheerio tgian nó nhanh vượt trội hơn + nó có hỗ trợ jquery lấy dễ dàng hơn ạ